src/locales/fr.js

```diff
@@ -63,7 +63,7 @@ export default {
     "peterAndPaulApostles": "Saint Pierre et Saint Paul, Apôtres",
     "presentationOfTheLord": "Présentation de Jésus au Temple",
     "sacredHeartOfJesus": "Sacré-Cœur de Jésus",
-    "theExaltationOfTheHolyCross": "The Exaltation of the Holy Cross",
+    "theExaltationOfTheHolyCross": "Exaltation de la Sainte-Croix",
```

Comment:
After verification in a missal and the [francophone bishop association](https://www.aelf.org/calendrier/romain/2019/09) website, I found that in French we don't literally translate it, but uses instead `La Croix Glorieuse`.